### PR TITLE
adapt to nodeqos and podqos api

### PR DESCRIPTION
**Pull request by mfanjie**
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: pkg/ensurance/executor/podinfo/pod_info.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	ElasticCPU                                                                      int64
+	PodCPUUsage, PodCPUShare, PodCPUQuota, PodCPUPeriod                             float64
+	ContainerCPUUsages, ContainerCPUShares, ContainerCPUQuotas, ContainerCPUPeriods []ContainerState
+
+	ActionType  ActionType
+	CPUThrottle CPURatio
+	Executed    bool
+}
+
+func ContainsPendingPod(pods []PodContext) bool {
```
Comment (chenkaiyue) [inline pkg/ensurance/executor/podinfo/pod_info.go:97]: Pending? What about ContainsNoExecutedPod?

File: pkg/resource/node_resource_manager.go [hunk 2]
```diff
@@ -130,7 +130,6 @@ func (o *NodeResourceManager) Run(stop <-chan struct{}) {
 			case state := <-o.stateChann:
 				o.state = state
 				o.lastStateTime = time.Now()
-			case <-tspUpdateTicker.C:
```
Comment (chenkaiyue) [inline pkg/resource/node_resource_manager.go:133]: Why delete this case ?

**Comment by chenkaiyue** [comment]
https://github.com/mfanjie/crane/pull/1

File: pkg/ensurance/executor/podinfo/pod_info.go (hunk 1 above)
Comment (mfanjie) [reply pkg/ensurance/executor/podinfo/pod_info.go:97]: pending means pending on actions to be executed, but it is confused cause pending generally means pending on scheduling in k8s, let me rename it.

File: pkg/resource/node_resource_manager.go (hunk 2 above)
Comment (mfanjie) [reply pkg/resource/node_resource_manager.go:133]: cause the origin logic is
1. collect the state from node by 10s interval
2.  compute node resource by 20s interval

The map is not synced, so the agent crashes frequently with `concurrent access to the map` error. 
By thinking deeper, I think node resource controller do not need to trigger the reconcile by interval, as long as there is new message from the channel, it should do reconcile. this simplifies  the flow.

**Review by chenkaiyue — APPROVED** [review]
lgtm

**Pull request merged**